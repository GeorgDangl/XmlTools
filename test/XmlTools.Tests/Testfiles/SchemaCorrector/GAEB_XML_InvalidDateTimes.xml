<?xml version="1.0" encoding="UTF-8"?>
<GAEB>
  <GAEBInfo>
    <Date>NotADate</Date>
    <Date>14-08-2018</Date>
  </GAEBInfo>
  <Award>
    <AwardInfo>
      <BidDate>0000-00-00</BidDate>
      <EvalEnd>0000-00-04</EvalEnd>
      <CnstStart>0000--34-33</CnstStart>
      <CnstEnd>2018-08-27</CnstEnd>
      <Cur>0000-00-00</Cur>
    </AwardInfo>
  </Award>
</GAEB>
