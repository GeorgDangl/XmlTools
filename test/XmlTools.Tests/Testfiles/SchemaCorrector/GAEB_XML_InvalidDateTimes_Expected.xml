<?xml version="1.0" encoding="UTF-8"?>
<GAEB>
  <GAEBInfo>
    <Date>2018-08-14</Date>
  </GAEBInfo>
  <Award>
    <AwardInfo>
      <CnstEnd>2018-08-27</CnstEnd>
      <Cur>0000-00-00</Cur>
    </AwardInfo>
  </Award>
</GAEB>
